src/less/summernote-bs4.less

```diff
@@ -151,6 +151,39 @@
         border-top: 1px solid @border-color;
       }
     }
+    .note-editor.note-frame .note-statusbar .note-status-output {
```

Comment:
@DiemenDesign `.note-editor.note-frame .note-statusbar` should be deleted in whole less/scss files. They are already declared in `.note-statusbar` class.

Reply:
+1